docs/analytics/quickstart.mdx

```diff
@@ -9,29 +9,39 @@ it is inefficient for analytical queries, which often scan a large amount of dat
 in a table.
 
 ParadeDB introduces special tables called `parquet` tables. These tables behave like regular Postgres tables but
-use a column-oriented layout via Apache Arrow and Parquet and leverage Apache DataFusion, a query engine
-optimized for column-oriented data.
+have two primary advantages:
 
-## Using Parquet Tables
+1. Significantly faster aggregate queries due to a column-oriented layout and query engine
+2. Lower disk space due to Parquet as the storage format
+
+## How to Use Parquet Tables
 
 ```sql
 -- USING parquet must be provided
 CREATE TABLE movies (name text, rating int) USING parquet;
 
+-- Insert and query data
 INSERT INTO movies VALUES ('Star Wars', 9), ('Indiana Jones', 8);
 SELECT AVG(rating) FROM movies;
 
-DROP TABLE movies;
+-- Clear the table
+TRUNCATE movies;
 ```
 
 That's it! `parquet` tables accept standard Postgres queries, so there's nothing new to learn.
 
+## When to Use Parquet Tables
+
+Because column-oriented storage formats are not designed for fast updates, `parquet` tables are for **append-only data**.
+`UPDATE` and `DELETE` clauses are not supported. Data that is frequently updated
+should be stored in regular Postgres `heap` tables.
```

Comment:
Maybe add a sentence to say that it can then be buffered to parquet tables on a cron job via `pg_cron`, which ships with the ParadeDB Docker image?

Reply:
I don't want to mix documentation about column-oriented tables with recommendations for database operations